### PR TITLE
[KO] Fix tutorial - quickstart

### PR DESCRIPTION
**Pull request by kimkun07**
Fix: Korean tutorial was wrong. with English tutorial

Remove softmax activation of the last Dense layer: Later the guide use loss function with from_logits=True parameter. softmax activation should not be included in `model`.
Delete compile part: Later the guide has compile part again.

### CODE REVIEW

**Comment by kimkun07** [comment]
I signed it!
